engine/src/main/java/org/terasology/rendering/opengl/FBO.java

```diff
@@ -358,9 +388,34 @@ private static void verifyCompleteness(String title, Type type, FBO fbo) {
         }
     }
 
+    /**
+     *  check if a texture is bound to the FBO with the identifier
+     * @param identifier
+     * @return
+     */
+    public  boolean IsTextureBound(String identifier)
```

Comment:
Methods usually start with lowercase. Also, `identifier` is somewhat unclear without context. Using an enum could help.